docs/sources/panels-visualizations/visualizations/canvas/index.md

```diff
@@ -55,9 +55,13 @@ Canvas introduces a new editing experience. You can now edit your canvas panel i
 
 Related to a fresh look at panel editing, the context menu enables you to perform common tasks quickly and efficiently. Supported functionality includes opening / closing the inline editor, duplicating an element, deleting an element, and more.
 
-The context menu is triggered by a right click action over the panel / over a given canvas element
+The context menu is triggered by a right click action over the panel / over a given canvas element. When right clicking the panel, you are able to set a background image and easily add elements to the canvas.
 
-{{< figure src="/static/img/docs/canvas-panel/canvas-context-menu-9-2-0.png" max-width="750px" caption="Canvas context menu" >}}
+{{< figure src="/static/img/docs/canvas-panel/canvas-panel-context-menu-9-3-0.png" max-width="750px" caption="Canvas panel context menu" >}}
```

Comment:
[/static/img/docs/canvas-panel/canvas-panel-context-menu-9-3-0.png](https://grafana.com/static/img/docs/canvas-panel/canvas-panel-context-menu-9-3-0.png) is not available. It appears on https://github.com/grafana/website/pull/10852, the image has an extra .png.